keras_hub/src/models/gemma3/gemma3_tokenizer_test.py

```diff
@@ -0,0 +1,63 @@
+import os
+
+import pytest
+
+from keras_hub.src.models.gemma3.gemma3_tokenizer import Gemma3Tokenizer
+from keras_hub.src.tests.test_case import TestCase
+
+
+class Gemma3TokenizerTest(TestCase):
+    def setUp(self):
+        self.init_kwargs = {
+            # Generated using create_gemma3_test_proto.py
+            # TODO: Uploaded the actual Gemma3 spm file, which is the wrong
+            # thing to do. Need to figure out how to add `<start_of_image>`,
+            # etc., tokens to the tokeniser during SPM training.
+            "proto": os.path.join(
+                self.get_test_data_dir(), "gemma3_test_vocab.spm"
+            )
+        }
+        # TODO: Figure out why <start_of_image>, etc. tokens are not getting
```

Comment:
Could be control symbols? https://github.com/google/sentencepiece/blob/master/doc/special_symbols.md which are meant to never surface string side, it's a sentencepiece thing.

Depends how the sentencepiece vocab was set up.

Reply:
Did not work

Reply:
As in, the Gemma3 SPM file decodes correctly,  but the trained one does not